### PR TITLE
Adds ParticipantsView & Route

### PR DESCRIPTION
**Pull request by novwhisky**
Thought I'd whip together a basic participants page

screen shot 2017-11-05 at 7 38 41 pm

### CODE REVIEW

File: src/components/Content/ParticipantList/Participant/Participant.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Participant = ({ username }) => {
+  return <div className="participant-ctn">
+    <div className="participant-avatar"></div>
+    <div className="participant-name">
+      {username}
```
Comment (elimisteve) [inline src/components/Content/ParticipantList/Participant/Participant.js:7]: @Moe-Shoman Let's make a User widget, or Member widget, and use it all over the place -- when we show users' usernames in ^^this^^, when users are mentioned in LeapChat (and we render the markdown to HTML that includes this widget), when a user is mentioned in the description/deliverables of a task, etc.

**Comment by novwhisky** [comment]
Just a quick prop warning fix

File: src/components/Content/ParticipantList/ParticipantList.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    getUsers();
+  }
+
+  renderList() {
+    const { users } = this.props;
+
+    return Object.keys(users).sort().reduce((acc, key) => {
+      const { username } = users[key];
+      return acc.concat(<Participant key={username} username={username} />)
+    }, []);
```
Comment (elimisteve) [inline src/components/Content/ParticipantList/ParticipantList.js]: @wittnl Would you mind just turning this into a for-loop?  It'll be simpler for people to read, and also use less memory in virtue of not allocating a function on every iteration.

**Comment by elimisteve** [comment]
@wittnl Very nice!  Later we can also show each user's karma points.

### Brainstorming Other Ideas

(I'm not saying these need to be added now, just brainstorming and trying to think ahead.)

One thing we will be storing in the database is _who invited whom_.  So having a hierarchical view -- that perhaps uses the same code as the Task Hierarchy view! -- of participants would also be awesome.

Toward the upper right we could have a drop-down that lets users select whether they want the hierarchy view or the list view.  And we should let people type (part of a) username and filter the list down based on that input.

**Comment by novwhisky** [comment]
Ok, I'll have to look further into the task heirarchy's recursivity and how those user relationships are modeled but here's the for loop change for now.

**Comment by elimisteve** [comment]
@wittnl Merged :tada: . Thank you :-D

**Pull request merged**